### PR TITLE
Auto deploy functionnality using grunt

### PR DESCRIPTION
**Pull request by piercus**
I have integrated with https://github.com/Tim-B/grunt-aws-lambda

Please share your feedbacks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,9 +13,14 @@ $ cp .env.sample .env
 Use the script to create your zip archive:
 
 ```
-$ npm run zip
+$ grunt run
```
Comment (TaylorBriggs) [inline README.md:17]: Can this be added as an alternative to the zip script + manual uploading to AWS? I'd like to document both options for folks that would rather upload their zip some other way.

File: gruntfile.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                  // Task-specific options go here.
+              }
+          }
+      }
+  });
+
+  grunt.registerTask('run', ['build_event','lambda_invoke']);
+
+  grunt.registerTask('deploy', ['run', 'build_event', 'lambda_package', 'lambda_deploy']);
+
```
Comment (TaylorBriggs) [inline gruntfile.js:49]: Would you remove the comments and from this file?

**Comment by TaylorBriggs** [comment]
Overall this looks good, just a couple minor formatting things I'd like to have cleaned up.

File: README.md (hunk 1 above)
Comment (piercus) [inline README.md:17]: Good remark

see 314d710 for the documentation.

I've also made a grunt task `grunt package` to just package the zip file.

**Comment by piercus** [comment]
Tell me if there is anything missing

**Comment by TaylorBriggs** [comment]
@piercus Thank you for making those changes. This looks great; I'll merge this in.

**Comment by piercus** [comment]
You're welcome, i'm glad to help !

**Pull request merged**